clients/src/test/java/org/apache/kafka/clients/consumer/internals/ConsumerNetworkThreadTest.java

```diff
@@ -327,8 +339,11 @@ void testEnsureEventsAreCompleted() {
         assertTrue(applicationEventsQueue.isEmpty());
     }
 
+    // Look into this one
```

Comment:
Can you elaborate on this comment?

Reply:
My mistake leaving that there, that was a comment for myself that I forgot to remove